nav2_motion_primitives/src/back_up.cpp

```diff
@@ -95,7 +95,7 @@ nav2_tasks::TaskStatus BackUp::controlledBackup()
 {
   auto current_pose = std::make_shared<geometry_msgs::msg::PoseWithCovarianceStamped>();
 
-  if (!robot_->getCurrentPose(current_pose)) {
+  if (!robot_->getGlobalLocalizerPose(current_pose)) {
```

Comment:
Rather than change all the actions to use the global localizer pose, lets leave them using the correct one. Then in the robot class, implement a getCurrentPose that just calls getGlobalLocalizerPose and add a TODO saying it needs to use the transform method found in costmap.

That way, we only have one place to change to use the correct pose method.

Maybe open an issue to remind us to implement that function as well.

Reply:
I can do that but we still need to change it in other places since the type will be different.